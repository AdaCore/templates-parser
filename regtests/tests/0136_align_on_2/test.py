from test_support import *

gprbuild('alo2')
run('alo2')
