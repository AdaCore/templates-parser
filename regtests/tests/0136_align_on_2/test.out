
1:----------------------------
void sync_sync_fct
      (void *a     , size_t a_len,
       void *result, size_t result_len);
2:----------------------------
void sync_sync_fct
      (void *a     , size_t a_len,
       void *result, size_t result_len);
3:----------------------------
       void *a     , size_t a_len
       void *result, size_t result_len

4:----------------------------

5:----------------------------
